mlflow/openai/utils.py

```diff
@@ -160,6 +160,11 @@ def __init__(self, api_type):
         self._is_azure_ad = api_type in ("azure_ad", "azuread")
         self._key_configured = bool(openai.api_key) or "OPENAI_API_KEY" in os.environ
```

Comment:
Can we remove `or "OPENAI_API_KEY" in os.environ`?

Reply:
Ah true, yeah no point in keeping it there.